internal/graph/check.go

```diff
@@ -675,7 +675,7 @@ ConsumerLoop:
 	return finalResult, nil
 }
 
-// checkUsersetSlowPath will check userset or public wildcard path.
+// checkUsersetSlowPath will check userset path.
```

Comment:
I don't understand why this comment was updated ? If I look at the code of `produceUsersetDispatches`, we still have the wildcard short-circuiting.

Reply:
The point of this PR is to have checkPublicAssignable instead of checkUsersetSlowPath checking against public wildcard.  In this way, [nestedUsersetFastPath](https://github.com/openfga/openfga/blob/af054d9693bd7ebd0420456b144c2fb6888aaf87/internal/graph/check.go#L1214) will still work when wildcard can be directly assigned to the relation.

Reply:
we can do it in separate PR later.